rakefile

```diff
@@ -13,4 +14,11 @@ end
 desc "Run bundler for local development and deployment"
 task :run_bundler do
     sh %{ bundle config unset deployment; bundle install; bundle config set deployment 'true'; bundle install }
-end
\ No newline at end of file
+end
+
+desc "Update lambda layers, environment_variables, vpc, and events"
+take :set_config do
```

Comment:
typo here for `rake`, source of the travis errors

Reply:
Thanks, fixed